slider/curve.py

```diff
@@ -0,0 +1,59 @@
+import math
+from .position import Position
+
+class Bezier():
+    def __init__(self, points):
+        self.points = points
+        self.xs = [i.x for i in points]
+        self.ys = [i.y for i in points]
+
+    def at(self, t):
+        def f(points, t):
```

Comment:
I think it might be more clear / faster to use the explicit definition of point on bezier curve instead of the recursive definition.